zerver/models.py

```diff
@@ -617,6 +617,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     # display settings
     twenty_four_hour_time = models.BooleanField(default=False) # type: bool
     default_language = models.CharField(default=u'en', max_length=MAX_LANGUAGE_ID_LENGTH) # type: Text
+    high_contrast_mode = models.BooleanField(default=False) # type: bool
```

Comment:
I believe `high_contrast_mode` should also be added to the UserProfile `property_types` dict in models.py so that it is automatically handled in other backend processes.